packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -274,78 +283,106 @@ const CreatePassword = ({
                         className='cfd-password-modal__content dc-modal__container_cfd-password-modal__body cfd-password-modal__create-password-content'
                         data-testid='dt_create_password'
                     >
-                        <Icon
-                            icon={platform === CFD_PLATFORMS.MT5 ? 'IcMt5OnePassword' : 'IcDxtradeOnePassword'}
-                            width='122'
-                            height='108'
-                        />
-                        <Text
-                            size='s'
-                            align='center'
-                            weight='bold'
-                            className='cfd-password-modal__create-password-title'
-                        >
-                            <Localize
-                                i18n_default_text='Create a {{platform}} password'
-                                values={{
-                                    platform: getCFDPlatformLabel(platform),
-                                }}
-                            />
-                        </Text>
-                        <Text size='xs' align='center' className='cfd-password-modal__create-password-description'>
-                            <Localize
-                                i18n_default_text='You can use this password for all your {{platform}} accounts.'
-                                values={{
-                                    platform: getCFDPlatformLabel(platform),
-                                }}
-                            />
-                        </Text>
-                        <div className='input-element'>
-                            <PasswordMeter
-                                input={values.password}
-                                has_error={!!(touched.password && errors.password)}
-                                custom_feedback_messages={getErrorMessages().password_warnings}
-                            >
-                                {() => (
-                                    <PasswordInput
-                                        autoComplete='new-password'
-                                        label={localize('{{platform}} password', {
+                        <div className='cfd-password-modal__create-password-body'>
+                            {platform === CFD_PLATFORMS.MT5 ? (
+                                <Icon icon={'IcMt5Password'} width='100' height='100' />
+                            ) : (
+                                <Icon icon={'IcDxtradeOnePassword'} width='122' height='108' />
+                            )}
+                            {platform !== CFD_PLATFORMS.MT5 ? (
```

Comment:
can we combine these conditions?
```suggestion
                            {platform !== CFD_PLATFORMS.MT5 ? (
                                    <Icon icon={'IcDxtradeOnePassword'} width='122' height='108' />
```

Reply:
Done...